core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1234,13 +1234,26 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    * driver can access the accumulator's `value`.
    */
   def accumulator[T](initialValue: T, name: String)(implicit param: AccumulatorParam[T])
-    : Accumulator[T] = {
+  : Accumulator[T] = {
     val acc = new Accumulator(initialValue, param, Some(name))
     cleaner.foreach(_.registerAccumulatorForCleanup(acc))
     acc
   }
 
   /**
+   * Create an [[org.apache.spark.Accumulator]] variable of a given type, with a name for display
```

Comment:
Should probably give this a different Scaladoc in order to differentiate it from the non-internal Accumulator and explain what's different.